### PR TITLE
BAP-58: fix ListBinding and TableBinding

### PR DESCRIPTION
**Pull request by afomina-quantori**
This pull request fixes the problem with usage of contains keyword when checking tables and rows.
Now we have two separate keywords:
* have - means list/table should have only the specified rows 
* contains - means that list/table should have at least the specified rows

### CODE REVIEW

**Review by SergeyRuzin — APPROVED** [review]

**Review by mikhail1aleksandrov1 — DISMISSED** [review]

File: BINDINGS.md [hunk 1]
```diff
@@ -201,9 +201,22 @@ Then the following controls should be empty:
...
+Then "Categories" should have the following items:
+
+* Check that list has given values in exact order
+[Then("(.+?) should have in exact order the following items:")]
+Example:
+Then "Categories" should have in exact order the following items:
+
+* Check that list contains given values
+[Given("(.+?) (has|does not have) the following items:")]
+[Then("(.+?) should (have|not have) the following items:")]
```
Comment (PavelAAlexeevQ) [inline BINDINGS.md:217]: wrong description, [Given("(.+?) (has|does not have) the following items:")] -> [Given("(.+?) (contains|does not contain) the following items:")] ?

**Review by PavelAAlexeevQ — COMMENTED** [review]
I can not see difference between _have_ and _contain_ . Will consumers of the package see the difference?

File: src/Behavioral.Automation/Bindings/TableBinding.cs [hunk 2]
```diff
@@ -1,105 +1,137 @@
...
+            else
+            {
+
+                Assert.ShouldBecome(() => gridRows.CellsText.ContainsValues(expectedValues),
+                    true,
+                    $"{gridRows.Caption} is {gridRows.CellsText.Aggregate((x, y) => $"{x}, {y}")}");
+            }
+        }
+
+        [Then("(.*?) should (have|not have) \"(.*)\" in (.*)")]
```
Comment (PavelAAlexeevQ) [inline src/Behavioral.Automation/Bindings/TableBinding.cs:84]: looks like no example for this binding in BINDINGS.md

**Review by PavelAAlexeevQ — COMMENTED** [review]
Lets suppose we have 1000 elements in row and would like to check only first 3 of them but with exact order. Previously we could write something
```
Then "Table" table contains the following rows in exact order:
1
2
3
```
Now we can't :(

File: src/Behavioral.Automation/Bindings/TableBinding.cs (hunk 2 above)
Comment (afomina-quantori) [reply src/Behavioral.Automation/Bindings/TableBinding.cs:84]: added description and example

**Review by SergeyRuzin — APPROVED** [review]

**Review by PavelAAlexeevQ — APPROVED** [review]

**Pull request merged**